files/nrpe/check_hwclock.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This script will check hwclock of the system.
+# Then compare hwclock with system time. if there
+# is more then 5 minutes of difference then it notify.
+
+## Nagios return codes
+WARNING=1
+CRITICAL=2
+UNKNOWN=3
+OK=0
+
+HWtime=$(sudo hwclock --debug | grep Hw | cut -d" " -f 8)
+SYStime=$(date +%s)
+let diff_time=$SYStime-$HWtime
+let diff_time_min=$diff_time/60
```

Comment:
IMO it is a good idea to use seconds as a time values, since people are mostly expecting it to be seconds.

Reply:
Suggested by @junousi 
https://github.com/CSCfi/puppet-opsviewagent/blob/master/files/nrpe/check_pci_ghosts.sh
and I have copied the concept from @martbhell

Reply:
Timing will be set back to seconds.